index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--header design-->
+    <header class="header">
+      <nav class="navbar">
+        <div class="logo-placeholder">
+          <h2 class="logo">My Logo</h2>
+        </div>
+        <div class="menu">
+        <img src="icons8-menu (1).svg" class="menubar">
+        </div>
+      </nav>
+    </header> 
+    <!--end of header design-->
+
+    <!--headline design-->
+    <section class="headline">
+      <div class="headline-div">
+        <h2 class="headline-1">I'm Joy. Glad to see you!</h2>
+      </div>
+
+      <div>
+        <p class="headline-2">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+      </div>
+
+      <div class="connect">
+        <h4>LET'S CONNECT</h4>
+      </div>
+
+      <div class="headline-images">
+        <img src="Vector.svg" alt="twitter" class="vector" />
+        <img src="Linkedin icon.svg" alt="Linkedin" class="vector" />
+        <img src="Vector (2).svg" alt="mail" class="vector" />
+        <img src="Vector (1).svg" alt="github" class="vector" />
+        <img src="Vector (3).svg" alt="hand" class="vector" />
+      </div>
```

Comment:
Great job on the list of social icons! However, there is a small adjustment that needs to be made to ensure proper linking functionality and semantic correctness.
- [ ] Instead of using a div tag for the list, please use a `<ul>` tag to group the social icons and each icon should be wrapped in an `<li>` tag. Additionally, make sure that each icon has a proper link (`<a>` tag) with the appropriate href attribute to connect the social networks.

Please make sure you update them. In this way, you will enhance the semantic structure of the social icons list and ensure that the icons are properly linked to their respective social networks.